### ISSUE TITLE
[Bug] Warning: Attempt to read property "namespaces" on null

### ISSUE THREAD

**Issue by salamichel** [synolia/SyliusSchedulerCommandPlugin#62]
**Composer infos**
```
sylius/calendar        v0.3.0  Date handling for PHP applications.
sylius/fixtures-bundle v1.7.0  Configurable fixtures for Symfony applications.
sylius/grid-bundle     v1.10.2 Amazing grids with support of filters and custom fields integrated into Symfony.
sylius/mailer-bundle   v1.6.0  Mailers and e-mail template management for Symfony projects.
sylius/paypal-plugin   v1.4.1  PayPal plugin for Sylius.
sylius/registry        v1.6.0  Services registry.
sylius/resource-bundle v1.8.4  Resource component for Sylius.
sylius/sylius          v1.11.4 E-Commerce platform for PHP, based on Symfony framework.
sylius/theme-bundle    v2.2.0  Themes management for Symfony projects.

synolia/sylius-scheduler-command-plugin v3.1.0 Scheduler Command Plugin.
```

**Describe the bug**
Try to click on "New" 

**To Reproduce**
1. click "Create"

**Screenshots**
```
ErrorException:
Warning: Attempt to read property "namespaces" on null
  at C:\wamp\www\sylius\Sylius-Standard\vendor\synolia\sylius-scheduler-command-plugin\src\Parser\CommandParser.php:63

```
![image](https://user-images.githubusercontent.com/3256040/164888606-25cc932f-51d9-4074-a02f-c30f815dd9aa.png)

**Additional context**

**Comment by salamichel** [comment]
... work correctly in "production" mode

**Comment by oallain** [comment]
Fixed in [v3.2.0](https://github.com/synolia/SyliusSchedulerCommandPlugin/releases/tag/v3.2.0).

Thanls @salamichel .
